smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/TypeScriptIntegration.java

```diff
@@ -234,29 +251,36 @@ default void addConfigInterfaceFields(
      *
      *     private static final Logger LOGGER = Logger.getLogger(CodegenVisitor.class.getName());
      *
-     *     public void addRuntimeConfigValues(
+     *     public Map<String, Consumer<TypeScriptWriter>> getRuntimeConfigWriters(
      *             TypeScriptSettings settings,
      *             Model model,
      *             SymbolProvider symbolProvider,
-     *             TypeScriptWriter writer,
      *             LanguageTarget target
      *     ) {
      *         // This is a static value that is added to every generated
      *         // runtimeConfig file.
-     *         writer.write("foo: 'some-static-value',"); // Note the trailing comma!
```

Comment:
The note about trailing commas, and the trailing commas themselves in the examples need to be re-added.